scipy/signal/tests/test_ltisys.py

```diff
@@ -411,12 +409,28 @@ def test_multioutput(self):
         assert_allclose(b_all, np.vstack((b0, b1, b2)), rtol=1e-13, atol=1e-14)
 
 
-class _TestLsimFuncs:
+class TestLsim:
     digits_accuracy = 7
 
-    @abstractmethod
     def func(self, *args, **kwargs):
-        pass
+        return lsim(*args, **kwargs)
```

Comment:
We had introduced `self.func` in order to switch between `lsim` & `lsim2` for shared tests. Now that there are no more two functions, we should remove this again, and do `self.func` -> `lsim` (i.e. undoing that aspect of https://github.com/scipy/scipy/commit/5acb9564989ea45706e1d175dfd7a49479d359f9)